lsp-mode.el

```diff
@@ -3611,6 +3613,9 @@ in that particular folder."
     (when lsp-enable-xref
       (add-hook 'xref-backend-functions #'lsp--xref-backend nil t))
 
+    (when lsp-headerline-breadcrumb-enable
```

Comment:
These should be added in `lsp-headerline-breadcrumb-mode`, to `lsp-configure-hook` and `lsp-unconfigure-hook`